### PR TITLE
Version 2.0

### PR DESCRIPTION
**Pull request by aattsai**
## What's does this do?

A new design and purpose. It's ongoing and requires a bit more work.

## How has this been tested?

Tested locally on mobile and desktop

Detailed changes are in commits below.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -37,65 +39,118 @@
     ga('send', 'pageview');
 
   </script>
+
   <link type="text/plain" rel="author" href="humans.txt" />
+  <link href="https://fonts.googleapis.com/css?family=Roboto:100,300,400" rel="stylesheet">
```
Comment (dijonkitchen) [inline index.html:44]: We should be able to pull this in from SUI.

File: index.html [hunk 2]
```diff
@@ -37,65 +39,118 @@
...
+  <div class="fifthSegment ui padded grid">
+    <div class="fourteen wide middle aligned centered column">
+      <h1>Pricing</h1>
+      <div class="ui three stackable centered aligned middle aligned cards">
+        <div class="card">
+          <div>
+            <img class="package-image" src="assets/prototype.png"/>
+          </div>
+          <h2>Prototype</h2>
+          <p>A proof of concept that you can interact</p>
```
Comment (dijonkitchen) [inline index.html]: Interact with?

File: index.html [hunk 3]
```diff
@@ -37,65 +39,118 @@
...
+          </div>
+          <h2>MVP+</h2>
+          <p>A user-heavy app aimed to impress</p>
+          <div class="ui horizontal divider"><i class="tag icon"></i></div>
+          <h3 class="price">
+            $ 20,000
+          </h3>
+        </div>
+      </div>
+      <button class="ui big red button build">Start building</button>
```
Comment (dijonkitchen) [inline index.html]: Link to contact form for now?

File: index.html [hunk 4]
```diff
@@ -104,55 +159,41 @@ <h1 class="ui header">
...
-        </i>
-        team
-      </h1>
-      <div class="team-photos ui three column grid">
-        <div class="profile-pic column">
-            <img id="aaron-profile-pic" class= "ui small centered circular image" src="assets/aaron.jpg" />
-            <p class="img-description">aaron<br>•<br>principal</p>
+  <div class="seventhSegment ui padded grid">
+    <div class="fourteen wide middle aligned centered column">
+      <form class="ui form" action="https://formspree.io/cortlandtlabs@gmail.com" method="POST">
```
Comment (dijonkitchen) [inline index.html]: Make a G Suite alias email to @aattsai ?

**Review by dijonkitchen — COMMENTED** [review]
Nice work @aattsai ! It looks great and it's perfectly responsive it seems! 

We still need work on content and pricing, but it's a good start.

File: index.html [hunk 5]
```diff
@@ -20,12 +20,14 @@
 
   <link rel="stylesheet" type="text/css" href="semantic/dist/semantic.min.css">
   <link rel="stylesheet" type="text/css" href="css/main.css">
+  <link rel="stylesheet" type="text/css" href="css/animate.css">
 
   <script defer src="https://code.jquery.com/jquery-3.1.0.js"   integrity="sha256-slogkvB1K3VOkzAI8QITxV3VzpOnkeNVsKvtkYLMjfk="   crossorigin="anonymous"></script>
```
Comment (dijonkitchen) [inline index.html]: I think we can use the minified version here too. It'll be a smaller file size and faster download / better human experience.

**Comment by dijonkitchen** [comment]
Might fix #30 too!

File: index.html (hunk 4 above)
Comment (aattsai) [reply index.html]: bd@cortlandtlabs.co then

Comment (dijonkitchen) [reply index.html]: No partnerships@cortlandtlabs.co?

Comment (aattsai) [reply index.html]: Productize it for now

Comment (aattsai) [reply index.html]: Partnerships@cortlandt can be for bigger projects

Comment (dijonkitchen) [reply index.html]: How about opportunities@... or ideas@...?

File: index.html [hunk 6]
```diff
@@ -22,7 +22,7 @@
   <link rel="stylesheet" type="text/css" href="css/main.css">
   <link rel="stylesheet" type="text/css" href="css/animate.css">
 
-  <script defer src="https://code.jquery.com/jquery-3.1.0.js"   integrity="sha256-slogkvB1K3VOkzAI8QITxV3VzpOnkeNVsKvtkYLMjfk="   crossorigin="anonymous"></script>
+  <script src="https://code.jquery.com/jquery-3.1.0.min.js" integrity="sha256-cCueBR6CsyA4/9szpPfrX3s49M9vUU5BgtiJj06wt/s=" crossorigin="anonymous"></script>
```
Comment (dijonkitchen) [inline index.html]: I think we can still defer loading the script to be async.

File: index.html [hunk 7]
```diff
@@ -37,65 +39,118 @@
...
-        <div class="work4"></div>
+  </div>
+  <div class="thirdSegment ui padded grid">
+    <div class="ten wide middle aligned centered column">
+      <h1>Technology</h1>
+      <div class="ui divider"></div>
+      <div class="ui three middle aligned cards">
+        <div class="card">
+          <div class="image">
+            <img src="assets/rails-logo.png" />
```
Comment (dijonkitchen) [inline index.html:84]: I think showing tech is that useful, but if we're going to put it, should we have a Ruby logo too?

Comment (aattsai) [reply index.html:84]: Rails would imply "Ruby" on Rails, no?

Comment (dijonkitchen) [reply index.html:84]: Yeah, so doesn't React imply JS?

Comment (aattsai) [reply index.html:84]: Oh that's a node.js logo

Comment (dijonkitchen) [reply index.html:84]: I think logos might look better without a border, but that's minor.

**Review by dijonkitchen — APPROVED** [review]
FYI, we need to confirm the FormSpree form after this is merged.

**Pull request merged**